### ISSUE TITLE
"docker-compose run" ends with "/usr/local/bin/python: No module named autogpt"

### ISSUE THREAD

**Issue by daudo** [Significant-Gravitas/Auto-GPT#3733]
### ⚠️ Search for existing issues first ⚠️

- [X] I have searched the existing issues, and there is no existing issue for my problem

### Which Operating System are you using?

Linux

### Which version of Auto-GPT are you using?

Latest Release

### GPT-3 or GPT-4?

GPT-3.5

### Steps to reproduce 🕹

preface: this may be related to #3403 

I was trying to follow the steps outlined in #3403 to get 0.2.2 up and running, which actually worked, but with 0.3.0 things went south again, albeit at a different step:

Following the documentation [1], here's what I did:

```
$ docker pull significantgravitas/auto-gpt
Using default tag: latest
latest: Pulling from significantgravitas/auto-gpt
[...]
Digest: sha256:62f84f7fef0b07ef01e22128da8e95d24cd1098c0336f164cc3c6eaca81042ea
Status: Image is up to date for significantgravitas/auto-gpt:latest
docker.io/significantgravitas/auto-gpt:latest
$ mkdir autogpt && cd autogpt
```

I then created a docker-compose.yml as described in the documentation:

```
$ cat docker-compose.yml
version: "3.9"
services:
  auto-gpt:
    image: significantgravitas/auto-gpt
    depends_on:
      - redis
    env_file:
      - .env
    environment:
      MEMORY_BACKEND: ${MEMORY_BACKEND:-redis}
      REDIS_HOST: ${REDIS_HOST:-redis}
    volumes:
      - ./:/app
    profiles: ["exclude-from-up"]
  redis:
    image: "redis/redis-stack-server:latest"
```

Based on https://github.com/Significant-Gravitas/Auto-GPT/blob/master/.env.template, I  created an .env file with my OpenAI API token an finally let docker-compose do its wonders:

```
$ docker-compose run --rm auto-gpt
[+] Running 2/2
 ✔ Network auto-gpt_default    Created                                                                                        0.0s 
 ✔ Container auto-gpt-redis-1  Created                                                                                        0.1s 
[+] Running 1/1
 ✔ Container auto-gpt-redis-1  Started                                                                                        0.3s 
/usr/local/bin/python: No module named autogpt

```

[1] https://docs.agpt.co/setup/#set-up-with-docker

### Current behavior 😯

auto-gpt doesn't start because the "autogpt" module can't be found

### Expected behavior 🤔

a nice and smooth and friendly chat with Auto-GPT :)

**Comment by resitan** [comment]
Similar issue here, just set everything up and got:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/app/autogpt/__main__.py", line 5, in <module>
    autogpt.cli.main()
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1635, in invoke
    rv = super().invoke(ctx)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/app/autogpt/cli.py", line 87, in main
    from autogpt.main import run_auto_gpt
  File "/app/autogpt/main.py", line 22, in <module>
    from scripts.install_plugin_deps import install_plugin_dependencies
ModuleNotFoundError: No module named 'scripts'
```

**Comment by hackern0v1c3** [comment]
Duplicate of #1513 
Looks like this is fixed in v 0.3.1 #3710

**Comment by daniel-alejandro-t** [comment]
If you use podman use:
`podman build --format=docker -t auto-gpt .
`
